robot_smach_states/src/robot_smach_states/manipulation/grab.py

```diff
@@ -129,7 +134,7 @@ def execute(self, userdata):
         else:
             rospy.loginfo("Updated pose of entity (dx, dy, dz) : (%f, %f, %f)" %
                           (updated_grab_entity.pose.position.x - grab_entity.pose.position.x,
```

Comment:
Yes, Entity.pose is a Python @property, that converts to a geometry_msg.Pose on the fly. the private Entity._pose is a PyKDL.Frame. 

There are a bunch of cases where this happens and I decided to not touch ALL of these in first instance. It's relatively easy to grep when this occurs so I think it saves a lot of time now to postpone this.